File: IronVault/IronVault.Model/Models/Seminar.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class Seminar
    {
        public int SeminarId { get; set; }

        public string Tema { get; set; } = null!;

        public string Predavac { get; set; } = null!;

        public DateTime Datum { get; set; }
    }
}

File: IronVault/UI/ironvault_mobile/lib/models/seminar.dart
import 'package:json_annotation/json_annotation.dart';

part 'seminar.g.dart';

@JsonSerializable()
class Seminar {
  int? seminarId;
  String? tema;
  String? predavac;
  DateTime? datum;

  Seminar({this.seminarId, this.tema});

  factory Seminar.fromJson(Map<String, dynamic> json) =>
      _$SeminarFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$SeminarToJson(this);
}
